manifests/beanstalkd_exporter.pp

```diff
@@ -105,6 +109,7 @@
   Boolean $manage_user                = true,
   String $os                          = $prometheus::os,
   String $extra_options               = '',
+  Optional[String] $proxy             = undef,
```

Comment:
can you please use `Optional[String[1]]`?

Reply:
Put the [1], dunno which one is better.